docs/manual/manual_telemetry.rst

```diff
@@ -0,0 +1,235 @@
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 8108 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 4006 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+	| Vertiq 2306 | .. centered:: |:x:|                | Servo modules do not support specialized telemetry messages such as those detailed here, but it is possible to access the same information using IQUART messages on servo modules. |
+	+-------------+------------------------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+
+Supported Telemetry
+=====================
+The structure of the supported telemetry messages and how to trigger them varies between different protocols. The sections below cover these details for different protocols.
+
+Hobby Protocols
+****************
+ESC telemetry is available when using :ref:`Hobby Protocols <manual_hobby>` with Vertiq modules. Telemetry messages are sent from the module as a serial
+message at the baud rate configured on the module.
+
+The exact method for requesting telemetry varies depending on what type of hobby protocol is being used, but the structure and content of 
+the telemetry sent is the same across all hobby protocols.
+
+Structure and Content
+######################
+The structure and content of the ESC telemetry message available when using :ref:`Hobby Protocols <manual_hobby>` is based on the `telemetry
+of KISS and BLHeli ESCs <https://www.rcgroups.com/forums/showatt.php?attachmentid=8524039&d=1450424877>`_. 
+
+**There is one key difference between the telemetry sent by Vertiq modules and the KISS standard. The KISS standard specifies sending ERpm data in the telemetry message. ERpm data

```

Comment:
Clarifying for users where we diverge from what is laid out for KISS ESCs on the RPM reporting

Reply:
FCs may convert eRPM to RPM by having the user specify motor pole pairs.  If they specify 1 pole pair, then eRPM and RPM are the same.  I think you should mention this.

Reply:
Ah, I see you addressed this elsewhere.  Instead, perhaps just provide a link to on of your two sections discussing this topic.
https://iqmotion.readthedocs.io/en/add-telemetry-information/tutorials/flight_controller_telemetry_tutorial.html#fc-telemetry-erpm-to-rpm

Reply:
I added links to the sections discussing PX4 and ArduCopter ERPM to RPM conversion to the note just below this on flight controller considerations